lib/Tvm.js

```diff
@@ -202,7 +229,7 @@ class Tvm {
       // appear in the I/O Runtime activation logs as an application error
       return {
         error: {
-          body: { error: 'server error' },
+          body: { error: 'server error ' + e.message },
```

Comment:
let's not expose any uncatched error message for security reasons to the user, either we catch it and know what to return to the user either we don't know and we should look at the logs.

Reply:
just show a generic error message.